R/revenuemetric.R

```diff
@@ -3,14 +3,14 @@
 #' @description Creates a small multiple plot by sub-groups
 #' @inherit RevenueData
 #' @param FUN A function that calculates a metric
```

Comment:
It would be helpful to list out the functions that could be used for the `FUN` parameter

Reply:
Thanks. I've incremented the version. It's a long time since I've done a commit!

I'm not going to implement the suggestion about documenting the FUN parameter. It's a sensible request, but one that would involve a lot of work which given that this is currently only for internal use feels unnecessary.